jira/resources.py

```diff
@@ -593,7 +593,7 @@ def __init__(
 
     def update(  # type: ignore[override] # incompatible supertype ignored
         self, dashboard_id: str, item_id: str, value: dict[str, Any]
-    ) -> Resource:
+    ) -> DashboardItemProperty:
```

Comment:
recall we need to update the corresponding docstring return type

Reply:
Ah! I thought I got all of those.